src/rickbox/database/database_helper.py

```diff
@@ -39,7 +39,7 @@
 
 import pymysql
 
-from database_config import MYSQL_HOST, MYSQL_USER, MYSQL_PASS, MYSQL_DB
+from database.database_config import MYSQL_HOST, MYSQL_USER, MYSQL_PASS, MYSQL_DB
```

Comment:
PyCharm was blowing up here and this fixed it. :shrug:

Reply:
It's because of how PyCharm handles modules. If you mark the directory as sources root it'll accomplish the same task of suppressing the warning. The Python compiler doesn't care either way.